### PR TITLE
Enhancement/datepicker support text

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: src/app/views/shared/services/library-menu.service.ts [hunk 1]
```diff
@@ -126,6 +126,9 @@ export class LibraryMenuService {
     }, {
       name: 'Buttons',
       link: LibraryItemSegmentPaths.buttons
+    }, {
+      name: 'Datepicker',
```
Comment (proand) [inline src/app/views/shared/services/library-menu.service.ts:130]: I think Datepicker should still be hidden until the Angular component is ready...
...and the value of `LibraryItemSegmentPaths.datepicker` should be Datepicker, not DatePicker

**Review by proand — APPROVED** [review]
See inline comment, otherwise ok 👍

File: src/app/views/shared/services/library-menu.service.ts [hunk 2]
```diff
@@ -169,6 +172,9 @@ export class LibraryMenuService {
     }, {
       name: 'Global header',
       link: LibraryItemSegmentPaths.globalheader
+    }, {
+      name: 'Date & time',
```
Comment (proand) [inline src/app/views/shared/services/library-menu.service.ts]: ...and don't show this either of course ;)

Comment (proand) [reply src/app/views/shared/services/library-menu.service.ts]: ...already done, my bad.

Comment (yleaxeman) [reply src/app/views/shared/services/library-menu.service.ts]: Same second :-)

**Pull request merged**